### PR TITLE
Update dashboard plugin guide - Closes 1858

### LINKED ISSUE #1858: Update dashboard plugin guide
### Description
A recent update in the dashboard plugin also exposes the registered endpoints from plugins. Update the content to describe how a plugin's endpoint can be invoked using the dashboard.

- Add information about how to enable the plugin using designated flags.
- Update the Hello application to the latest beta tag in `lisk-sdk-examples`.
- Invoke plugin endpoint using the dashboard

#### Affected branch / URL
`development`

#### Page location
`build-blockchain/using-dashboard.adoc

### PR DESCRIPTION
**Pull request by TalhaMaliktz**
Closes #1858

### CODE REVIEW

File: docs/modules/ROOT/pages/build-blockchain/using-dashboard.adoc [hunk 1]
```diff
@@ -90,12 +96,46 @@ After updating the *custom_config.json* file, build the blockchain client again:
...
++
+[source,bash]
+----
+lisk-core start --network testnet --enable-dashboard-plugin
 ----
+--
+=====
+====
+
+If you registered the Dashboard via `plugins.ts`, then:
```
Comment (chris529) [inline docs/modules/ROOT/pages/build-blockchain/using-dashboard.adoc]:
```suggestion
If you registered the Dashboard via `plugins.ts`, then perform the following:
```

File: docs/modules/ROOT/pages/build-blockchain/using-dashboard.adoc [hunk 2]
```diff
@@ -163,13 +201,18 @@ image::guides/dashboard/recent_transactions.png["Recent transactions",750,align=
...
 In the previous guides, the following endpoints were created:
 
+.*For Module:*
 . The endpoint `hello_getHello` is for xref:{url_guides_module_getHello}[Getting the latest Hello for an address].
-. The endpoint `hello_getHelloCounter` is for xref:{url_guides_module_getHelloCounter}[Getting the Hello message counter].
+
+.*For Plugin:*
+. The endpoint `helloInfo_getMessageList` is for xref:{url_guides_plugin_getMessageList}[Getting a list of all Hello messages sent on-chain].
+
+Let's see, how to invoke each endpoint:
```
Comment (chris529) [inline docs/modules/ROOT/pages/build-blockchain/using-dashboard.adoc]: Let's see, how to invoke each endpoint: 

imho I would suggest we slightly reword this sentence to the following:
How to invoke each endpoint is covered in the following section:

File: docs/modules/ROOT/pages/build-blockchain/using-dashboard.adoc [hunk 3]
```diff
@@ -186,31 +229,34 @@ image::guides/dashboard/call_action.png["Invoke hello_getHello",750,align="cente
...
 At the bottom of the Dashboard is the *Recent events* window, which allows the possibility to subscribe to various events by selecting them from a dropdown box.
 
 Select the `hello_newHello` event.
 
-.Subscribe to the newHello event
+.Subscribe to the 'newHello' event
 image::guides/dashboard/subscribe_event.jpg["Subscribe to newHello event", 1000,align="center"]
 
-Once a new hello transaction is posted, the newHello event is also published subsequently and will show up in the *Recent events* window.
+Once a new hello transaction is posted, the `newHello` event is also published subsequently and will show up in the *Recent events* window.
```
Comment (chris529) [inline docs/modules/ROOT/pages/build-blockchain/using-dashboard.adoc]:
```suggestion
Once a new hello transaction is posted, the `newHello` event is also published, and will subsequently appear in the *Recent events* window.
```

File: docs/modules/ROOT/pages/build-blockchain/using-dashboard.adoc [hunk 4]
```diff
@@ -228,20 +274,4 @@ For more information, see {url_typedoc_crypto}[getLisk32AddressFromAddress^].
...
-
-// You can directly click on the kbd:[Submit] button to view the results.
-// Again, it is not necessary to provide any input, because the action doesn't require any.
-
-// image::guides/dashboard/call_pluginaction.png["Invoke plugin action",400]
-
-// As expected, the plugin responds with the latest hello message that was posted to the blockchain application.
-
-// image::guides/dashboard/call_pluginaction_success.png["Plugin action success",400]
+You can also see general node and network information, etc through the Dashboard plugin.
```
Comment (chris529) [inline docs/modules/ROOT/pages/build-blockchain/using-dashboard.adoc]:
```suggestion
It is also possible to view the general node and network information, etc., through the Dashboard plugin.
```

**Review by chris529 — COMMENTED** [review]
Looks great, just a few minor suggestions 👍

File: docs/modules/ROOT/pages/build-blockchain/using-dashboard.adoc (hunk 2 above)
Comment (TalhaMaliktz) [reply docs/modules/ROOT/pages/build-blockchain/using-dashboard.adoc]:
```suggestion
To invoke an endpoint through the Dashboard, refer to the following section:
```

**Review by chris529 — APPROVED** [review]
LGTM 👍

**Review by Incede — APPROVED** [review]

**Pull request merged**